text/morph.js

```diff
@@ -188,6 +188,9 @@ export class Text extends Morph {
       },
 
       padding: {
+        spec: {
```

Comment:
We might do that but I think for now its fine (and more user friendly) to keep it simple.